packages/dds/tree/src/test/feature-libraries/sequence-field/rebase.test.ts

```diff
@@ -230,22 +230,30 @@ export function testRebase() {
 
 		it("remove ↷ earlier remove", () => {
 			// Removes ---DE
-			const removeA = Change.remove(3, 2);
+			const removeA = Change.remove(3, 2, tag1);
 			// Removes AB--
-			const removeB = Change.remove(0, 2);
+			const removeB = Change.remove(0, 2, tag2);
 			const actual = rebase(removeA, removeB);
 			// Removes -DE
-			const expected = [Mark.tomb(tag1, brand(0), 2), Mark.skip(1), Mark.remove(2, brand(0))];
+			const expected = [
+				Mark.tomb(tag2, brand(0), 2),
+				Mark.skip(1),
+				Mark.remove(2, brand(0), { revision: tag1 }),
+			];
```

Comment:
The `rebase` function defaults to `tag1` for the `baseRev` argument when it's not provided, which is awkward for this test because `tag1` is the rebased change's tag.
```suggestion
			// Removes ---DE
			const removeA = Change.remove(3, 2, tag2);
			// Removes AB--
			const removeB = Change.remove(0, 2, tag1);
			const actual = rebase(removeA, removeB);
			// Removes -DE
			const expected = [
				Mark.tomb(tag1, brand(0), 2),
				Mark.skip(1),
				Mark.remove(2, brand(0), { revision: tag2 }),
			];
```